Teamspeak LXC Alpine install error

### ✅ Have you read and understood the above guidelines?

yes

### 📜 What is the name of the script you are using?

Teamspeak-Server

### 📂 What was the exact command used to execute the script?

bash -c "$(curl -fsSL https://raw.githubusercontent.com/community-scripts/ProxmoxVE/main/ct/alpine-teamspeak-server.sh)"

### ⚙️ What settings are you using?

- [x] Default Settings
- [ ] Advanced Settings

### 🖥️ Which Linux distribution are you using?

Ubuntu 24.04

### 📝 Provide a clear and concise description of the issue.

RELEASE=$(curl -fsSL https://teamspeak.com/en/downloads/#server | sed -n '/teamspeak3-server_linux_amd64-/ { s/.*teamspeak3-server_linux_amd64-\([0-9]\+\.[0-9]\+\.[0-9]\+\).*/\1/p; q }')
curl: (23) Failure writing output to destination

### 🔄 Steps to reproduce the issue.

1. run `bash -c "$(curl -fsSL https://raw.githubusercontent.com/community-scripts/ProxmoxVE/main/ct/alpine-teamspeak-server.sh)"`
2. choose Verbose mode

### ❌ Paste the full error output (if available).

    ___    __      _                ______                    _____                  __        _____                          
   /   |  / /___  (_)___  ___      /_  __/__  ____ _____ ___ / ___/____  ___  ____ _/ /__     / ___/___  ______   _____  _____
  / /| | / / __ \/ / __ \/ _ \______/ / / _ \/ __ `/ __ `__ \\__ \/ __ \/ _ \/ __ `/ //_/_____\__ \/ _ \/ ___/ | / / _ \/ ___/
 / ___ |/ / /_/ / / / / /  __/_____/ / /  __/ /_/ / / / / / /__/ / /_/ /  __/ /_/ / ,< /_____/__/ /  __/ /   | |/ /  __/ /    
/_/  |_/_/ .___/_/_/ /_/\___/     /_/  \___/\__,_/_/ /_/ /_/____/ .___/\___/\__,_/_/|_|     /____/\___/_/    |___/\___/_/     
        /_/                                                    /_/                                                            
  ⚙️  Using Default Settings on node pve (🔍  Verbose)
  🆔  Container ID: 115
  🖥️  Operating System: alpine (3.22)
  📦  Container Type: Unprivileged
  💾  Disk Size: 2 GB
  🧠  CPU Cores: 1
  🛠️  RAM Size: 256 MiB
  🔍  Verbose Mode: Enabled
  🚀  Creating a Alpine-TeamSpeak-Server LXC using the above default settings
  
  ✔️   Validated Storage (rootdir / vztmpl).
  ✔️   LXC Template List Updated
  ✔️   Started LXC Container
  ✔️   Customized LXC Container
  ✔️   Set up Container OS
  ✔️   Network Connected: 192.168.50.104
  ✔️   Internet Connected
  ✔️   DNS Resolved github.com to 140.82.121.4
  ⏳   Updating Container OSfetch http://dl-cdn.alpinelinux.org/alpine/latest-stable/main/x86_64/APKINDEX.tar.gz
fetch http://dl-cdn.alpinelinux.org/alpine/latest-stable/community/x86_64/APKINDEX.tar.gz
(1/7) Upgrading openrc-user (0.62.3-r0 -> 0.62.6-r0)
(2/7) Upgrading openrc (0.62.3-r0 -> 0.62.6-r0)
Executing openrc-0.62.6-r0.post-upgrade
(3/7) Upgrading alpine-release (3.22.0-r0 -> 3.22.1-r0)
(4/7) Upgrading libcrypto3 (3.5.0-r0 -> 3.5.1-r0)
(5/7) Upgrading ca-certificates-bundle (20241121-r2 -> 20250619-r0)
(6/7) Upgrading libssl3 (3.5.0-r0 -> 3.5.1-r0)
(7/7) Upgrading alpine-base (3.22.0-r0 -> 3.22.1-r0)
Executing busybox-1.37.0-r18.trigger
OK: 35 MiB in 72 packages
  ✔️   Updated Container OS
  ⏳   Installing dependenciesfetch http://dl-cdn.alpinelinux.org/alpine/latest-stable/main/x86_64/APKINDEX.tar.gz
fetch http://dl-cdn.alpinelinux.org/alpine/latest-stable/community/x86_64/APKINDEX.tar.gz
(1/6) Installing ca-certificates (20250619-r0)
(2/6) Installing musl-obstack (1.2.3-r2)
(3/6) Installing libucontext (1.3.2-r0)
(4/6) Installing gcompat (1.1.0-r4)
(5/6) Installing libgcc (14.2.0-r6)
(6/6) Installing libstdc++ (14.2.0-r6)
Executing busybox-1.37.0-r18.trigger
Executing ca-certificates-20250619-r0.trigger
OK: 38 MiB in 78 packages
  ✔️   Installed dependencies
curl: (23) Failure writing output to destination, passed 1369 returned 0

[ERROR] in line 23: exit code 23: while executing command RELEASE=$(curl -fsSL https://teamspeak.com/en/downloads/#server | sed -n '/teamspeak3-server_linux_amd64-/ { s/.*teamspeak3-server_linux_amd64-\([0-9]\+\.[0-9]\+\.[0-9]\+\).*/\1/p; q }')


[ERROR] in line 1281: exit code 0: while executing command lxc-attach -n "$CTID" -- bash -c "$(curl -fsSL https://raw.githubusercontent.com/community-scripts/ProxmoxVE/main/install/"$var_install".sh)" $?


### 🖼️ Additional context (optional).

_No response_

--- a/ct/alpine-teamspeak-server.sh
+++ b/ct/alpine-teamspeak-server.sh
@@ -27,7 +27,7 @@ function update_script() {
     exit 1
   fi
 
-  RELEASE=$(curl -fsSL https://teamspeak.com/en/downloads/#server | sed -n '/teamspeak3-server_linux_amd64-/ { s/.*teamspeak3-server_linux_amd64-\([0-9]\+\.[0-9]\+\.[0-9]\+\).*/\1/p; q }')
+  set +o pipefail && RELEASE=$(curl -fsSL https://teamspeak.com/en/downloads/#server | sed -n 's/.teamspeak3-server_linux_amd64-([0-9]+.[0-9]+.[0-9]+)./\1/p' | head -1) && set -o pipefail
 
   if [ "${RELEASE}" != "$(cat ~/.teamspeak-server)" ] || [ ! -f ~/.teamspeak-server ]; then
     msg_info "Updating ${APP} LXC"

--- a/install/alpine-teamspeak-server-install.sh
+++ b/install/alpine-teamspeak-server-install.sh
@@ -20,7 +20,7 @@ $STD apk add --no-cache \
   libc6-compat
 msg_ok "Installed dependencies"
 
-RELEASE=$(curl -fsSL https://teamspeak.com/en/downloads/#server | sed -n '/teamspeak3-server_linux_amd64-/ { s/.*teamspeak3-server_linux_amd64-\([0-9]\+\.[0-9]\+\.[0-9]\+\).*/\1/p; q }')
+set +o pipefail && RELEASE=$(curl -fsSL https://teamspeak.com/en/downloads/#server | sed -n 's/.teamspeak3-server_linux_amd64-([0-9]+.[0-9]+.[0-9]+)./\1/p' | head -1) && set -o pipefail
 
 msg_info "Installing Teamspeak Server v${RELEASE}"
 mkdir -p /opt/teamspeak-server

